internal/tool/generate/generator.go

```diff
@@ -880,6 +881,14 @@ func (g *generator) generateImports(p printFn) {
 	p(`)`)
 }
 
+func (g *generator) generateVersionCheck(p printFn) {
+	p(`var _ %s = %s[[%d][%d][%d]int](%q)`,
+		g.codegen().qualify("LatestVersion"), g.codegen().qualify("Version"),
+		codegen.Major, codegen.Minor, codegen.Patch,
```

Comment:
Should we avoid complaining about patch number changes? Those changes should be for bug fixes. If there is a compatibility change, we should expect at least a minor version bump?